test-app/app/components/group-list-item.hbs

```diff
@@ -0,0 +1 @@
+<ul>{{yield}} elephant</ul>
```

Comment:
```suggestion
<ul>{{yield}} elephant</ul>

```

Just to add a new line at the end of file

Reply:
I think this is prevented by `.editoconfig`:

```toml
[*.hbs]
insert_final_newline = false
```

Do you think we should flip this setting?

Reply:
Ah my bad! No that's fine, I just think our addons are not consistent regarding this config. You can ignore my other comments regarding this 👌